### PR TITLE
Remove sprockets support

### PR DESCRIPTION
**Pull request by gmq**
Over the years, Rails has been transitioning away from sprockets for frontend purposes. Maintaining support for both methods of installation has added complexity to every update we make.

This PR removes sprockets support, leaving only support for the Webpacker installation of ActiveAdmin. Although Webpacker is deprecated, it's easy to move files to make Addons work with other frontend bundlers.

It also removes some custom fixes for the Active Material theme.

### CODE REVIEW

File: lib/generators/activeadmin_addons/install/install_generator.rb [hunk 1]
```diff
@@ -11,7 +11,8 @@ def setup_assets
         if use_webpacker?
           generate "activeadmin_addons:webpacker"
         else
-          generate "activeadmin_addons:assets"
+          puts "ActiveAdmin Addons requires ActiveAdmin installed with webpacker. "\
```
Comment (monkeyci) [inline lib/generators/activeadmin_addons/install/install_generator.rb:14]: **[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Do not write to stdout. Use Rails's logger if you want to log.

File: lib/generators/activeadmin_addons/install/install_generator.rb [hunk 2]
```diff
@@ -11,7 +11,8 @@ def setup_assets
...
+          "Please run 'rails generate active_admin:install --webpacker' and then 'rails generate activeadmin_addons:install'"
```
Comment (monkeyci) [inline lib/generators/activeadmin_addons/install/install_generator.rb:15]: **[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is too long. [125/100]

File: Rakefile [hunk 3]
```diff
@@ -18,29 +18,14 @@ task :prepare_assets do
...
-end
-
-task :webpack_tests do
-  system "rake prepare_assets"
-  system "export SPROCKETS=false; rspec ./spec/features"
-end
-
-task :all_tests do
-  system "rake prepare_assets"
   system "rake tests"
```
Comment (ldlsegovia) [inline Rakefile:26]: I think this task is not necessary now. Better: `rspec ./spec/lib`

File: spec/dummy/config/initializers/active_admin.rb [hunk 4]
```diff
@@ -331,5 +331,5 @@
   # By default, Active Admin uses Sprocket's asset pipeline.
   # You can switch to using Webpacker here.
   #
-  config.use_webpacker = ENV["SPROCKETS"] != "true"
+  config.use_webpacker = "true"
```
Comment (ldlsegovia) [inline spec/dummy/config/initializers/active_admin.rb:334]: `true` instead `"true"`

File: spec/support/active_admin_helpers.rb [hunk 5]
```diff
@@ -74,7 +74,7 @@ def load_resources(reset_app = true)
       # it allover the place
       ActiveAdmin.application.authentication_method = false
       ActiveAdmin.application.current_user_method = false
-      ActiveAdmin.application.use_webpacker = ENV["SPROCKETS"] != "true"
+      ActiveAdmin.application.use_webpacker = "true"
```
Comment (ldlsegovia) [inline spec/support/active_admin_helpers.rb:77]: same `true` instead `"true"`

**Review by ldlsegovia — COMMENTED** [review]
👏🏻

**Review by difernandez — CHANGES_REQUESTED** [review]

**Review by ldlsegovia — APPROVED** [review]

**Review by difernandez — APPROVED** [review]
Perfect 💯

**Pull request merged**